pkg/authorization/workspace_content_authorizer.go

```diff
@@ -89,23 +88,10 @@ func (a *workspaceContentAuthorizer) Authorize(ctx context.Context, attr authori
 		return authorizer.DecisionNoOpinion, "empty or system workspace", nil
 	}
 
-	isServiceAccount := validation.IsServiceAccount(attr.GetUser())
-	isAuthenticated := sets.New[string](attr.GetUser().GetGroups()...).Has("system:authenticated")
-	isForeign := validation.IsForeign(attr.GetUser(), cluster.Name)
-	isInScope := validation.IsInScope(attr.GetUser(), cluster.Name)
+	isServiceAccount := rbacregistryvalidation.IsServiceAccount(attr.GetUser())
+	isInScope := rbacregistryvalidation.IsInScope(attr.GetUser(), cluster.Name)
 
 	if IsDeepSubjectAccessReviewFrom(ctx, attr) {
-		attr := deepCopyAttributes(attr)
```

Comment:
just to recap, why we removing this "fallback t" anonymous?

Reply:
It's automaticly done now by the downstream authorizer.